### ISSUE TITLE
Example for segmentation?

### ISSUE THREAD

**Issue by andife** [Visual-Behavior/detr-tensorflow#19]
Hello,
since detr can also be used for panoptic segmentation I wonder if there are plans to add an example for this as well?
Thank you

**Comment by thibo73800** [comment]
Yes, we do have a version of panoptic detr internally, we'll probbaly add it to the repository soon.

**Comment by andife** [comment]
Would be great!

**Comment by BeusenBart** [comment]
Hi, do you still have any plans to share the code for the panoptic of instance segmentation detr? I've been working on the segmentation modules in tensorflow myself, based on the officical pytorch implementations, and it would be great to compare. For now, I don't see a segmentation.py in you repository.
